backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/VmHandler.java

```diff
@@ -1435,10 +1435,16 @@ public void autoSelectDefaultDisplayType(Guid srcEntityId,
 
         if (defaultDisplayType == null) {
             if (!displayGraphicsSupport.isEmpty()) {// when not found otherwise, let's take osinfo's record as the default
-                Map.Entry<DisplayType, Set<GraphicsType>> entry = displayGraphicsSupport.entrySet().iterator().next();
-                defaultDisplayType = entry.getKey();
```

Comment:
why not leaving it as this, just take the first entry, and change the order for the default/current CL in osinfo?
It would be more clear and predictable than these conditions below